public/app/plugins/panel/heatmap-new/HeatmapPanel.tsx

```diff
@@ -99,9 +99,16 @@ export const HeatmapPanel: React.FC<HeatmapPanelProps> = ({
     const { min, max } = reduceField({ field, reducers: [ReducerID.min, ReducerID.max] });
     const display = field.display ? (v: number) => formattedValueToString(field.display!(v)) : (v: number) => `${v}`;
 
+    let hoverValue: number | undefined = undefined;
+    if (hover && info.heatmap.fields) {
+      const countField = info.heatmap.fields[2];
+      const countVals = countField?.values.toArray();
+      hoverValue = countVals?.[hover.index];
```

Comment:
```suggestion
      hoverValue = countField?.values..get(hover.index);
```
Can simplify to use the data frame index access